src/ParallelAlgorithms/Events/ObserveNorms.hpp

```diff
@@ -154,12 +181,20 @@ class ObserveNorms<ObservationValueTag, tmpl::list<ObservableTensorTags...>,
   static constexpr Options::String help =
       "Observe norms of tensors in the DataBox.\n"
       "\n"
+      "You can choose the norm type for each observation. Note that the\n"
```

Comment:
Can you document that the integral norm won't work with subcell or FD?

Reply:
Done